### DISCUSSION TITLE
Way to pass data in the wasi-experimental workload?

### DISCUSSION THREAD

**Discussion by ReneeGA2020** [dotnet/runtime#97542] [Q&A]
I know that  wasi-experimental workload is not officially supported.

Just here to ask if anyone have any insight on this:

I have a wasmtime build by .net, any chance I can use it to host a   wasi-wasm console project created by the wasi-experimental workload (or mono runtime) and invoke functions inside by passing an int array as parameters?

It looks like there were some samples on github around invoking c# in wasi, but they seems to all about the old wasi-sdk version, which has much slower performance than the wasi-wasm version.

**Comment by maraf** [comment]
Do I understand it correctly that you are using wasmtime as a library inside of a .NET app and you want to run a second .NET app compiled as wasi-wasm inside of that wasmtime, right?
